Bootstrap/Setup.php

```diff
@@ -926,4 +926,12 @@ private function getMaxStateId()
 
         return 0;
     }
+
+    private function addIndexToChangeTable()
```

Comment:
Please check Setup::createMyTables function. You don't need this. Just add your index in initial schema creation.